### PR TITLE
HuggingFace pytorch-pretrained-bert GPT

### PR DESCRIPTION
**Pull request by VictorSanh**
Hello,
Following yesterday's PR for BERT models, here's the integration for GPT.
Tell me anything changes are needed!

I'll also do the integration for GPT2 and Transformer-XL probably later today or early next week.
Thanks again!
Victor

ping @ailzhang

### CODE REVIEW

File: huggingface_pytorch-pretrained-bert_gpt.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+model.eval()
+
+# Predict hidden states features for each layer
+with torch.no_grad():
+	hidden_states = model(tokens_tensor)
+```
+
+We can predict the next tokens using `openAIGPTLMHeadModel`.
+
+```python
```
Comment (ailzhang) [inline huggingface_pytorch-pretrained-bert_gpt.md]: Hey @VictorSanh , is it possible to deduplicate code blocks and only leave this full script? I'm thinking about moving the sentences into comments if it's not much. What do you think?

File: huggingface_pytorch-pretrained-bert_gpt.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+---
+layout: pytorch_hub_detail
+background-class: pytorch-hub-background
+body-class: pytorch-hub
+title: GPT
+summary: GPT models
+category: research
```
Comment (ailzhang) [inline huggingface_pytorch-pretrained-bert_gpt.md]: We've renamed this label to `researchers` ;) It'd great if you can keep it consistent. Thanks!

File: huggingface_pytorch-pretrained-bert_gpt.md (hunk 1 above)
Comment (ailzhang) [reply huggingface_pytorch-pretrained-bert_gpt.md]: @VictorSanh Also I got `predicted_token = '.</w>'` when I ran the model, is it expected behavior?

Comment (VictorSanh) [reply huggingface_pytorch-pretrained-bert_gpt.md]: Yes! it is the expected behavior (I made it clearer). The '</w>' is for the BPE encoding. You can use the `decode` method in the `tokenizer` to decode from the sequence of token ids.

Comment (VictorSanh) [reply huggingface_pytorch-pretrained-bert_gpt.md]: Yes, that's true, it's a bit repetitive, I moved everything in the same example snippet.

Comment (ailzhang) [reply huggingface_pytorch-pretrained-bert_gpt.md]: @VictorSanh  would you mind sending another PR to deduplicate the BERT file as well? Thanks a lot!

Comment (VictorSanh) [reply huggingface_pytorch-pretrained-bert_gpt.md]: Yes, that's what I was doing. 😁
Have similar duplications in the BERT examples.
